src/chart/pie/PieView.ts

```diff
@@ -268,12 +272,43 @@ class PieView extends ChartView {
             group.add(sector);
         }
 
+
+        const animationModel = seriesModel.isAnimationEnabled() ? seriesModel : null;
+        const drawBackground = seriesModel.get('showBackground', true);
+        const basicOuterRadius = getBasicPieLayout(seriesModel, api).r;
+        const bgStyle = seriesModel.getModel('backgroundStyle').getItemStyle();
```

Comment:
If `data.backgroundStyle` is not supported, how should developers set each pie piece to be a different color other than the default colors? I would suggest supporting `data.backgroundStyle` but not `data.showBackground` (only `series.showBackground` is supported).